react-main/src/index.js

```diff
@@ -1,10 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import './scss/index.scss';
-import registerServiceWorker from './registerServiceWorker';
+import Game from './app/screens/Game';
```

Comment:
You are importing the screen. You should be importing src/app instead, since that component will include the routing logic and more stuff.